pkg/protocols/http/request.go

```diff
@@ -486,10 +486,6 @@ func (request *Request) executeTurboHTTP(input *contextargs.Context, dynamicValu
 
 // ExecuteWithResults executes the final request on a URL
 func (request *Request) ExecuteWithResults(input *contextargs.Context, dynamicValues, previous output.InternalEvent, callback protocols.OutputEventCallback) error {
-	if request.Pipeline || request.Race && request.RaceNumberRequests > 0 || request.Threads > 0 {
-		variablesMap := request.options.Variables.Evaluate(generators.MergeMaps(dynamicValues, previous))
-		dynamicValues = generators.MergeMaps(variablesMap, dynamicValues, request.options.Constants)
-	}
```

Comment:
These had to go.

It was originally introduced in f3de611b for #2110, extended with `Constants` in 0d2d5106, and then refactored into `Make()` in 45cc676f. The issue is that the early eval broke expressions like `"{{dsl_func('{{runtime_var}}')}}"` and kept re-evaluating `rand_*()` for every request in flow templates.

We now pass `dynamicValues` properly (1224e701) to keep the context intact. Everything now uses `Make()` for consistent runtime evaluation.

Covered by integration tests (ab368910 & 80bd753c) btw.